lib/screens/explore_screens/unplant_seeds/interactor/mappers/amount_changer_mapper.dart

```diff
@@ -22,10 +23,10 @@ class AmountChangerMapper extends StateMapper {
         TextEditingValue(text: quantity, selection: TextSelection.fromPosition(TextPosition(offset: quantity.length)));
 
     return currentState.copyWith(
+      pageCommand: UpdateTextController(newAmountController),
       onFocus: false,
       unplantedInputAmount: tokenAmount,
       unplantedInputAmountFiat: fiatAmount,
-      controller: TextEditingController.fromValue(newAmountController),
```

Comment:
@gguij004 I thought you remove this controller from the bloc

Reply:
I add this controller in the UI and this assignation to a listener, could you move this code to the listener in other PR?

```
    final TextEditingValue newAmountController =
        TextEditingValue(text: quantity, selection: TextSelection.fromPosition(TextPosition(offset: quantity.length)));
```

Reply:
Yes, I made a note to do the change. Thanks